book-crawler/tests/test_async_api.py

```
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+"""
+비동기 API 테스트 스크립트
+"""
+
+import requests
+import time
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)